src/screens/Messages/Conversation/MessageInput.web.tsx

```diff
@@ -118,7 +145,23 @@ export function MessageInput({
         // @ts-expect-error web only
         onMouseEnter={() => setIsHovered(true)}
         onMouseLeave={() => setIsHovered(false)}>
+        <Button
+          onPress={e => {
+            e.currentTarget.measure((_fx, _fy, _width, _height, px, py) => {
+              openEmojiPicker?.({top: py, left: px, right: px, bottom: py})
+            })
+          }}
+          style={[a.p_xs, {marginTop: 3}]}
+          label={_(msg`Open emoji picker`)}
+          accessibilityHint={_(msg`Open emoji picker`)}
+          variant="ghost"
```

Comment:
small nit, can we make the background color of this match the background color of the input? or maybe just some color of gray? it looks kinda loud on bright mode

![Screenshot 2024-09-06 at 2 57 22 PM](https://github.com/user-attachments/assets/aa24b721-6304-4095-afc6-7f5d4a6e4c94)

Reply:
Sure I'll try some things